### ISSUE TITLE
batgrep manpage typo

### ISSUE THREAD

**Issue by HarHarLinks** [eth-p/bat-extras#59]
![image](https://user-images.githubusercontent.com/2803622/124497877-d2314800-ddbb-11eb-9af3-42a4460d4fc6.png)

I had a look at the repo, not sure where this originates.
```
Name            : bat-extras-git
Version         : 2021.04.06.r7.g20cdf5a-1
```
from AUR, which is current master.

**Comment by HarHarLinks** [comment]
ironic to make a typo in a typo issue title

**Comment by eth-p** [comment]
`\fR\fIbat\fRfor` would be rendered as `batfor`. It looks like I need to fix the manual generator.

**Comment by eth-p** [comment]
Fixed in ef0356a2d8da91241a3e435f5c4c03ad5d2442cb.
